packages/daemon/src/guest.js

```diff
@@ -67,7 +63,7 @@ export const makeGuestMaker = ({
       selfFormulaIdentifier: guestFormulaIdentifier,
       specialNames: {
         SELF: guestFormulaIdentifier,
-        HOST: hostFormulaIdentifier,
+        HOST: hostHandleFormulaIdentifier,
```

Comment:
We should add a test verifying that the guest actually can't do anything as the host now that this change is made.